src/status_im/contexts/preview/quo/avatars/collection_avatar.cljs

```diff
@@ -7,7 +7,11 @@
     [status-im.contexts.preview.quo.preview :as preview]))
 
 (def descriptor
-  [{:key     :image
+  [{:key     :size
+    :type    :select
+    :options [{:key :size-24}
+              {:key :size-20}]}
```

Comment:
What is the value of keywordizing the size? Can't it just be `24` and `20` so that it can be used directly in component's style above without mapping it

Reply:
as commented above - link to team decision etc  https://github.com/status-im/status-mobile/pull/17279#issuecomment-1721616168